scripts/jenkins/cloud/manual/run-deployer.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
name of file perhas 'run-containerized-deployer.sh' ?  or run-docker-deployer.sh?  its a nit pick but "deployer" is pretty overloaded.

Reply:
Good point :smile: I changed the name.